pkg/api/subscription.go

```diff
@@ -22,11 +22,32 @@ const (
 	SubscriptionStateWarned       SubscriptionState = "Warned"
 	SubscriptionStateSuspended    SubscriptionState = "Suspended"
 	SubscriptionStateDeleted      SubscriptionState = "Deleted"
+
+	// SubscriptionFeatureE2E is the feature in the subscription that is used
+	// to indicate if we need to save ARO cluster config into the E2E StorageAccount
+	SubscriptionFeatureE2E string = "Microsoft.ContainerService/SaveAROTestConfig"
 )
 
 // SubscriptionProperties represents subscription properties
 type SubscriptionProperties struct {
 	MissingFields
 
-	TenantID string `json:"tenantId,omitempty"`
+	TenantID           string                        `json:"tenantId,omitempty"`
+	AccountOwner       AccountOwnerProperties        `json:"accountOwner,omitempty"`
+	RegisteredFeatures []RegisteredFeatureProperties `json:"registeredFeatures,omitempty"`
+}
+
+// AccountOwnerProperties represents the subscription account owner informations
+type AccountOwnerProperties struct {
+	MissingFields
+
+	Email string `json:"email,omitempty"`
```

Comment:
Why does this need to be saved in the RP? ARM anyway scrubs PI of the customer before sending to the RP because of GDPR. In case some of that info flow through by mistake we shouldn't be saving it (encrypted or not).